projects/elife.yaml

```diff
@@ -350,6 +350,7 @@ elife-bot:
             "{instance}-elife-published":
                 deletion-policy: retain
                 public: true
+                cors: True
```

Comment:
appears everything is supported http://yaml.org/type/bool.html but better to be consistent